source/general/time.rst

```diff
@@ -71,15 +71,18 @@ In addition to that main way of specifying times as a floating point number wrt.
 the following header keys with date and time values as strings can be added.
 This is for convenience and humans reading the information. Usually science tools will not access
 this redundant and optional information. The time system used should be the one given by ``TIMESYS``.
+All values for the keywords mentioned here shall be ISO8601 date and time strings,
+``yyyy-mm-ddTHH:MM:SS.fff``.
+The precision is not fixed, so seconds or fractional seconds could be left out.
 
 * ``DATE-OBS`` type: string
-    * Observation start date (format: "yyyy-mm-dd")
-* ``TIME-OBS`` type: string
-    * Observation start time (format: "hh:mm:ss.sss...")
+    * Observation date and time, typically the start of observations if not defined otherwise in the comment
+* ``DATE-BEG`` type: string
+    * Observation start date and time
+* ``DATE-AVG`` type: string
+    * Average observation date and time
 * ``DATE-END`` type: string
-    * Observation end date (format: "yyyy-mm-dd")
-* ``TIME-END`` type: string
-    * Observation end time (format: "hh:mm:ss.sss...")
+    * Observation end date and time
 
 Note that the FITS standard allows and it is quite common to instead put a
```

Comment:
Shouldn't this sentence be updated as well then?

Reply:
yes